src/kernel/filesystem/vfs.zig

```diff
@@ -74,16 +74,16 @@ pub const FileSystem = struct {
     /// Arguments:
     ///     IN self: *const FileSystem - The filesystem in question being operated on
     ///     IN node: *const FileNode - The file being read from
-    ///     IN len: usize - The number of bytes to read from the file
+    ///     IN bytes: []u8 - The buffer to fill data from the file with
     ///
-    /// Return: []u8
-    ///     The data read as a slice of bytes. The length will be <= len, including 0 if there was no data to read
+    /// Return: usize
+    ///     The length of the actuall data read. This being < bytes.len is not considered an error.
```

Comment:
We should also mention that it's impossible for the returned value to be greater than the len. Same for the Write type.

Reply:
Added